tests/pages/components/conversations/conversation-part.js

```diff
@@ -11,6 +11,14 @@ export default {
     scope: '[data-test-body]'
   },
 
+  closedAt: {
+    scope: '[data-test-closed-at]'
+  },
+
+  isComment: hasClass('conversation-part--comment'),
+  isClosed: hasClass('conversation-part--closed'),
+  isByCurrentUser: hasClass('conversation-part--is-self'),
```

Comment:
I saw a question that got marked as outdated, which caused me to think page objects might not be completely clear to you. They are only ever used in tests and they are defined in a way that provides us with information we need to answer questions in tests.

If a conversation  thread lists multiple conversations of different `partType` values, we want to ask

- is this part displayed as a comment
- is this other part displayed as "user closed at"

We know that a comment has a `conversation-part--coment` class, and we know that the other one has a `conversation-part--closed` class, so we could ask

```Javascript
assert.ok($('conversation-part').classNames().indexOf('conversation-part--comment') > -1);
assert.ok($('conversation-part').classNames().indexOf('conversation-part--closed') > -1);
```

and that works, except it's not always completely clear what it is we're actually asking. Also, in multiple tests, we tend to ask the same or similar questions, so we have to repeat all of that. Then, if, for example, the class name changes, we have to change it in all of those places.

What page objects allow us to do is to specify

```Javascript
isClosed: hasClass('conversation-part--closed'),
```

in just one place and use that.

```Javascript
assert.ok(conversationPart.isClosed);
assert.ok(conversationPart.isComment);
```

That makes the question much clearer. Also, if the class changes, we change it in just that one place - in the page object.